adhocracy4/comments_async/static/comments_async/comment_form.jsx

```diff
@@ -111,13 +112,13 @@ export default class CommentForm extends React.Component {
           </form>
         </div>
       )
-    } else if (!this.props.isReadOnly) {
+    } else if (!isAuthenticated && !this.props.isReadOnly) {
```

Comment:
We don't need !isAuthenticated here because that is clear from the if before. But it can stay to be more explicit. :)